client/app/components/dashboards/CreateDashboardDialog.jsx

```diff
@@ -8,6 +8,7 @@ import { wrap as wrapDialog, DialogPropType } from "@/components/DialogWrapper";
 import navigateTo from "@/components/ApplicationArea/navigateTo";
 import recordEvent from "@/services/recordEvent";
 import { policy } from "@/services/policy";
+import { urlForDashboard } from "@/services/dashboard";
```

Comment:
How about we transform the response we receive and add a URL field to it, so we don't have to import (or remember to use) `urlForDashboard` everywhere?